### PR TITLE
Implement 'Cite' for datasets

### PR DESCRIPTION
**Pull request by egonw**
Implements #2325

### Description
Adds the `Cite` button to the `dataset` aspect.
    
### Caveats

> Please list anything which has been left out of this PR or which should be considered before this PR is accepted
Check any of the following which apply:

* [ ]  Breaking change (fix or feature that would cause existing functionality to not work as expected)
* [ ]  This change requires a documentation update
    * [ ]  I have made corresponding changes to the documentation
* [ ]  This change requires new dependencies (please list)

*If you make changes to the Python code*
  
* [x]  My code passes the [tox](https://tox.readthedocs.io/en/latest/) check, I can receive warnings about tests, documentation or both

### Testing

* https://scholia.toolforge.org/dataset/Q120320769
* https://scholia.toolforge.org/dataset/Q120320769/export

### Checklist
* [ ] I have [commented](https://peps.python.org/pep-0008/#comments) my code, particularly in hard-to-understand areas
* [x] My changes generate no new warnings
* [x] I have not used code from external sources without attribution
* [ ] I have considered [accessibility](https://www.w3.org/standards/webdesign/accessibility) in my implementation 
* [x] There are no remaining debug statements (print, console.log, ...)

### CODE REVIEW

**Review by larsgw — APPROVED** [review]

**Comment by fnielsen** [comment]
The curation link disappears `<a id='curation-link' role="button" class="btn btn-outline-secondary d-none">Improve data</a  I think it is the `page_actions` that does that.

**Comment by larsgw** [comment]
Oops, I missed that. Yes, I think it needs a `{{ super() }}` before `endblock`, that's what the work template has.

**Comment by egonw** [comment]
I'm on it.

File: scholia/app/templates/dataset.html [hunk 1]
```diff
@@ -12,6 +12,9 @@
 
 {% endblock %}
 
+{% block page_actions %}
+<a href="/dataset/{{ q }}/export" role="button" class="btn btn-outline-secondary">Cite</a>
+{% endblock %}
```
Comment (egonw) [inline scholia/app/templates/dataset.html:17]:
```suggestion
{{ super() }}
{% endblock %}
```

**Pull request merged**